packages/@aws-cdk/custom-resources/lib/aws-custom-resource/aws-custom-resource.ts

```diff
@@ -209,6 +221,14 @@ export class AwsCustomResource extends cdk.Construct implements iam.IGrantable {
       }
     }
 
+    for (const call of [props.onCreate, props.onUpdate, props.onDelete]) {
+      if (call && call.physicalResourceId?.responsePath) {
```

Comment:
```suggestion
      if (call?.physicalResourceId?.responsePath) {
```
since you're already using optional chaining on this line...

Reply:
Yeap 👍